### PR TITLE
Use global styles from psammead

### LINKED ISSUE #3718: Use @psammead-styles/global-styles in Simorgh
**Is your feature request related to a problem? Please describe.**
Once `@psammead-styles/global-styles` is created and published (https://github.com/bbc/psammead/pull/2021), use it in Simorgh. 

**Describe the solution you'd like**
- Use it in Simorgh 
  - delete `lib/globalStyles/*`
  - `import GlobalStyles from '@bbc/psammead-styles/global-styles';`
- Use it in Storybook setup - `.storybook/config.js`

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Ensure there are no visual regressions. Check that Font loading, normalize CSS and box sizing CSS is working as in the live application. 

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #3718 

**Overall change:** _use global-styles defined in `@psammead-styles`_

**Code changes:**

- _Bump `@psammead-styles` and pull `GlobalStyles` from `@psammead-styles`_
- _Update Storybook setup - `.storybook/config.js` to use imported GlobalStyles_
- _Remove global-styles from `app/lib/`_
- _Update snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/Layouts/defaultPageWrapper.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { node } from 'prop-types';
+import GlobalStyle from '@bbc/psammead-styles/global-styles';
```
Comment (sareh) [inline src/app/Layouts/defaultPageWrapper.jsx]: For consistency with other usage of this import. 
```suggestion
import GlobalStyles from '@bbc/psammead-styles/global-styles';
```

File: src/app/Layouts/defaultPageWrapper.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 import HeaderContainer from '../containers/Header';
 import FooterContainer from '../containers/Footer';
 import ManifestContainer from '../containers/Manifest';
 import ServiceWorkerContainer from '../containers/ServiceWorker';
-import GlobalStyle from '../lib/globalStyles';
 import MPulseBeacon from '../containers/MPulseBeacon';
 
 const PageWrapper = ({ children }) => (
```
Comment (sareh) [inline src/app/Layouts/defaultPageWrapper.jsx]: We'll need to pass in the fonts into the GlobalStyles component.

```
import GlobalStyles from '@bbc/psammead-styles/global-styles';
import { ServiceContext } from '../contexts/ServiceContext';
...
const { fonts } = useContext(ServiceContext);
...
<GlobalStyles fonts={fonts} />
```

**Review by sareh — CHANGES_REQUESTED** [review]
The Storybook update works well. 
For the applicaiton, we need to pass in the fonts to the GlobalStyles component.

**Review by sareh — APPROVED** [review]
Great work

✅  Fonts are still loading (checked on News and Persian articles
✅  Normalize still loading
✅  Box-sizing border-box still loading
http://localhost.bbc.com:7080/news/articles/c0g992jmmkko
Screenshot News article

http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o
Screenshot Persian article

**Review by dr3 — APPROVED** [review]

**Review by jroebu14 — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**